app/assets/stylesheets/layouts/_header.scss

```diff
@@ -1,4 +1,10 @@
 .header {
   background-color: $color-body;
   text-align: center;
+  margin-top: 40px;
+}
+
+.keg-picture {
+  width: 25%;
```

Comment:
You might want to use their grid now for layout http://getbootstrap.com/css/#grid

Leaving this is fine too.